analysis/SiemensQuadraProperties.py

```diff
@@ -1,24 +1,29 @@
 import math
 
+import PhysicsConstants as PC
+
 def DetectorRadius():
   return 400 # 82cm radius - 1cm crystal half-depth. Maybe 82 should be inner radius and need to adjust simulation?
 
 def CrystalVolume():
   return 0.32 * 0.32 * 2.0
 
-def CrystalMass():
-  return CrystalVolume() * 7.4
+def CrystalMass( DetectorMaterial ):
+  varName = f'density_{DetectorMaterial}'
+  if varName not in PC.__dict__:
+    raise RuntimeError(f'{varName} does not exist in PhysicsConstants. Wrong material name?')
+  return CrystalVolume() * eval(f'PC.{varName}')
```

Comment:
As before - isn't there a better way to do this?

Reply:
I saved densities in PhysicsConstants.py as a dictionary and get the density value from it based on a key which is a string with crystal name. Hopefully this simplifies the code now.